execution_engine_testing/tests/src/test/regression/gh_1902.rs

```diff
@@ -0,0 +1,158 @@
+use casper_engine_test_support::{
+    internal::{
+        DeployItemBuilder, ExecuteRequestBuilder, InMemoryWasmTestBuilder,
+        DEFAULT_ACCOUNT_PUBLIC_KEY, DEFAULT_RUN_GENESIS_REQUEST,
+    },
+    AccountHash, DEFAULT_ACCOUNT_ADDR, MINIMUM_ACCOUNT_CREATION_BALANCE,
+};
+use casper_execution_engine::{
+    core::engine_state::ExecuteRequest,
+    shared::system_config::auction_costs::{DEFAULT_ADD_BID_COST, DEFAULT_DELEGATE_COST},
+};
+use casper_types::{
+    account::Account,
+    runtime_args,
+    system::{
+        auction::{self, DelegationRate},
+        mint, standard_payment,
+    },
+    Gas, PublicKey, RuntimeArgs, SecretKey, U512,
+};
+use once_cell::sync::Lazy;
+
+const BOND_AMOUNT: u64 = 42;
+const DELEGATE_AMOUNT: u64 = 100;
+const DELEGATION_RATE: DelegationRate = 10;
+
+static ACCOUNT_1_SECRET_KEY: Lazy<SecretKey> =
+    Lazy::new(|| SecretKey::secp256k1_from_bytes([99; 32]).unwrap());
+static ACCOUNT_1_PUBLIC_KEY: Lazy<PublicKey> =
+    Lazy::new(|| PublicKey::from(&*ACCOUNT_1_SECRET_KEY));
+static ACCOUNT_1_ADDR: Lazy<AccountHash> = Lazy::new(|| AccountHash::from(&*ACCOUNT_1_PUBLIC_KEY));
+
+fn setup() -> InMemoryWasmTestBuilder {
+    let mut builder = InMemoryWasmTestBuilder::default();
+    builder.run_genesis(&*DEFAULT_RUN_GENESIS_REQUEST);
+    let id: Option<u64> = None;
```

Comment:
Why not just put `Option::<u64>::None` below directly?

Reply:
Consistency - all of the testing code uses this pattern, so I went with it.